### PR TITLE
Fix space-opera tests

### PR DESCRIPTION
**Pull request by elalish**
Reverts #1631 
Reverts #1381 

There's still one error, but not in a specific test, so we need to figure that out too.

### CODE REVIEW

File: packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts [hunk 1]
```diff
@@ -25,16 +25,13 @@ export const styles: CSSResult = css`:host {
...
   overflow: hidden;
   padding: 0 20px;
-  transition-duration: 300ms;
-  transition-timing-function: ease-out;
 }
 
 .SectionContent[open] {
   background-color: var(--me-theme-container-background);
-  overflow: hidden;
+  overflow: unset;
```
Comment (stevesan) [inline packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts:32]: These changes don't seem related. Are they in the original PR?

**Review by stevesan — APPROVED** [review]

File: packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts (hunk 1 above)
Comment (elalish) [reply packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts:32]: No, that's from reverting chris' recent PR, which broke some tests because they weren't being run.

**Comment by elalish** [comment]
Closing in favor of #1636
